### ISSUE TITLE
New composer design

### ISSUE THREAD

**Issue by jounih** [lensterxyz/lenster#808]
![Image](https://user-images.githubusercontent.com/10109867/195556559-4d27dbbd-6b61-4511-8f92-86680c625ecb.png)

![Image](https://user-images.githubusercontent.com/10109867/195556483-1ee49ca5-55f9-44c4-bd45-0e59e43c32d6.png)

![Image](https://user-images.githubusercontent.com/10109867/195556710-0234672a-2735-49f6-9cf5-4ca488865c4d.png)

**Comment by bigint** [comment]
Moving to next milestone 🙇🏼

**Comment by jounih** [comment]
niice! Will try to finish the other composer tabs for you for early next week (and also design for token gating collects via Lit 👀)

**Comment by en0c-026** [comment]
Can I connect to the server running the instance for development?
 I need to solve that in order to be able to modify the necessary components and test them in the browser. I spent all day trying to configure an AWS EC2 and the application runs well on the testnet but crashes after a minute and I have to stop the instance. 
 
 It would be great to be able to log in your dev server from my SSH terminal.

**Comment by nitegeist** [comment]
@bigint I noticed that the only finished component for this design is the audio tab. Any idea if or when there will be others? It would be good to have an idea of what the other tabs are supposed to look like before starting any dev work on this.

The flow I'm getting so far from the designs is that these icons can be selected and would probably bring up their respective tabs in the modal, while the "what's happening?" section would still bring up the publication tab.
![image](https://user-images.githubusercontent.com/31413091/230821669-82cce720-7603-42c6-bf9e-1b806fd4c9f1.png)

Correct me if I'm wrong @jounih. Also are you still working on the rest of the composer tabs? If they are done, would you mind posting them here? If not, I'd like to help with the design as well as the development.

**Comment by bigint** [comment]
Issue is part of #539

**Comment by Min2who** [comment]
Can i work on it?

**Comment by rajdip-b** [comment]
@Min2who hey, would you be up for collab?

**Comment by rishi-raj-jain** [comment]
@rajdip-b I just saw that you were working on #4169

**Comment by rajdip-b** [comment]
> @rajdip-b I just saw that you were working on #4169

yeah, i aim to primarily take up that one, would like to collab on this once since it's a bigger issue.

**Comment by rishi-raj-jain** [comment]
> > @rajdip-b I just saw that you were working on #4169
> 
> yeah, i aim to primarily take up that one, would like to collab on this once since it's a bigger issue.

Cool, all the best!

**Comment by rajdip-b** [comment]
@rishi-raj-jain would you be up for a collab?

**Comment by rishi-raj-jain** [comment]
Oh nah, I'm already working on #4091

**Comment by Min2who** [comment]
> @Min2who hey, would you be up for collab?

up for it

**Comment by rajdip-b** [comment]
> > @Min2who hey, would you be up for collab?
> 
> up for it

Great!! please drop me you discord so i can ping you there

**Comment by rajdip-b** [comment]
> Oh nah, I'm already working on #4091

thats cool, maybe another time

**Comment by rishi-raj-jain** [comment]
> > Oh nah, I'm already working on #4091
> 
> thats cool, maybe another time

For sure!

**Comment by Min2who** [comment]
> > > @Min2who hey, would you be up for collab?
> > 
> > 
> > up for it
> 
> Great!! please drop me you discord so i can ping you there

min2_dev

**Comment by rajdip-b** [comment]
@Min2who check your DM please!

**Comment by Min2who** [comment]
@bigint any timeline for draft pr

**Comment by rajdip-b** [comment]
hey @bigint, can you please send the design link? like figma or something? or, if it's not present, can you send the screenshots of every tab in the composer?

**Comment by bigint** [comment]
@rajdip-b we need to wait couple days to freeze the design internally, in the meanwhile can you work on tiptap one, cuz both are dependent 🙇🏼

**Comment by Min2who** [comment]
> 2 days @Min2who 🙇🏼 

Then this?

**Comment by rajdip-b** [comment]
> > 2 days @Min2who 🙇🏼
> 
> Then this?

think that might be after the design is fixed. @bigint, can we be assigned this issue? so that other folks dont get confused?

**Comment by sc0rp10n-py** [comment]
hey @jounih 
Is this bounty still open for assignment?
